x-pack/legacy/plugins/apm/public/components/shared/Links/apm/ErrorOverviewLink.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { APMLink, APMLinkExtendProps } from './APMLink';
+import { useUrlParams } from '../../../../hooks/useUrlParams';
+import { pickKeys } from '../../../../utils/pickKeys';
+import { APMQueryParams } from '../url_helpers';
+
+interface Props extends APMLinkExtendProps {
+  serviceName: string;
+  query?: APMQueryParams;
+}
+
+const ErrorOverviewLink = ({ serviceName, query, ...rest }: Props) => {
+  const { urlParams } = useUrlParams();
+
+  const persistedFilters = pickKeys(
+    urlParams,
+    'transactionResult',
+    'host',
+    'containerId',
+    'podName'
+  );
```

Comment:
These are now duplicated:
https://github.com/elastic/kibana/blob/9cd2bfd6d013c4f847ccb37185e864a809c6f34b/x-pack/legacy/plugins/apm/public/components/app/ErrorGroupOverview/index.tsx#L76

I don't think that's a problem now, but in the future we could lift the `filterNames` to be defined in the route config (where this linking function could also be defined).

Reply:
Yeah this is one of the downsides of loose coupling. But hopefully we can fix the routing soon

Reply:
> But hopefully we can fix the routing soon

++